app/jobs/reports/authentication_report.rb

```diff
@@ -0,0 +1,42 @@
+require 'reporting/authentication_report'
+
+module Reports
+  class AuthenticationReport < BaseReport
+    REPORT_NAME = 'authentication-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date)
+      return unless IdentityConfig.store.s3_reports_enabled
```

Comment:
We don't have it set in rubocop, maybe we should, but we do empty lines after returns about 70% of the time based on a quick scan through the code. I prefer the empty line to further differentiate the return.

Reply:
ah yeah agree! thank you!

Reply:
updated in https://github.com/18F/identity-idp/pull/9295/commits/7a4d364073c14456c22e727a73715ea15080d4c0